RoadMap_UIKit/RoadMap_UIKit/MainScreenVC.swift

```diff
@@ -0,0 +1,155 @@
+        email.frame = CGRect(x: 50, y: 340, width: 300, height: 30)
+        return email
+    }()
+
+    let passwordLabel: UILabel = {
+        let password = UILabel()
+        password.text = "Password"
+        password.textColor = .systemCyan
+        password.font = .systemFont(ofSize: 15, weight: .semibold)
+        password.frame = CGRect(x: 50, y: 400, width: 100, height: 50)
+        return password
+    }()
+
+    let passwordTextField: UITextField = {
+        let password = UITextField()
+        password.textColor = .black
+        password.font = .systemFont(ofSize: 13, weight: .light)
+        password.frame = CGRect(x: 50, y: 440, width: 300, height: 30)
+        password.isSecureTextEntry = true
+        return password
+    }()
+    
+    private lazy var eyeButton: UIButton = {
+        let button = UIButton()
+        button.setBackgroundImage(UIImage(systemName: "eye.fill"), for: .normal)
+        button.tintColor = .lightGray
+        button.addTarget(self, action: #selector(showHidePassword), for: .touchUpInside)
+        button.frame = CGRect(x: 300, y: 440, width: 40, height: 25)
+        return button
+    }()
+
+    let faceIdEntranceLabel: UILabel = {
+        let faceID = UILabel()
+        faceID.text = "Вход по FaceID"
+        faceID.font = .systemFont(ofSize: 15, weight: .semibold)
+        faceID.textColor = .black
+        faceID.frame = CGRect(x: 150, y: 510, width: 160, height: 50)
+        return faceID
+    }()
+
+    let faceIdSwitch: UISwitch = {
+        let faceSwitch = UISwitch()
+        faceSwitch.frame = CGRect(x: 280, y: 520, width: 60, height: 40)
+       return faceSwitch
+    }()
+
+    private lazy var enterButton: UIButton = {
+        let enter = UIButton()
+        enter.setTitle("Войти", for: .normal)
+        enter.backgroundColor = .systemCyan
+        enter.frame = CGRect(x: 100, y: 680, width: 200, height: 50)
+        enter.layer.cornerRadius = 10
+        enter.addTarget(self, action: #selector(toTheFriendsList), for: .touchUpInside)
+        return enter
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        emailTextField.setUnderLine()
+        passwordTextField.setUnderLine()
+        addSubviews()
+        view.backgroundColor = .white
+    }
+
+    func addSubviews() {
+        view.addSubview(appNameLabel)
+        view.addSubview(signInLabel)
+        view.addSubview(emailLabel)
+        view.addSubview(emailTextField)
+        view.addSubview(passwordLabel)
+        view.addSubview(passwordTextField)
+        view.addSubview(faceIdEntranceLabel)
+        view.addSubview(faceIdSwitch)
+        view.addSubview(enterButton)
+        view.addSubview(eyeButton)
+    }
+    
+    @objc func toTheFriendsList() {
+        let friendsList = FriendsListVC()
+        friendsList.title = "Birthday"
+        self.navigationController?.pushViewController(friendsList, animated: true)
+    }
+    
+    @objc func showHidePassword() {
```

Comment:
```suggestion
    @objc func showHidePasswordAction() {
```
 и перписать на guard